runtime_builders/publish_builders.py

```diff
@@ -59,38 +58,21 @@ def main():
                       .format(ke))
 
 
-def _format_and_write(project_name, latest, prefix, manifest):
-    parts = os.path.splitext(latest)
-    if parts[1] != '.yaml':
-        logging.error('Please provide yaml config file to publish as latest!')
-        sys.exit(1)
-    try:
-        project_prefix = prefix + project_name + '-'
-        latest_file = parts[0][len(project_prefix):]
-        full_latest_file = latest[len(prefix):]
-        if project_name not in manifest.keys():
-            manifest[project_name] = {}
-            manifest[project_name]['target'] = {}
-            manifest[project_name]['target']['file'] = None
-            # TODO: add logic here for 'file' vs 'runtime' when
-            # we add support for versioning
-        m_project = manifest[project_name]
-        prev_builder = m_project['target']['file']
-        if prev_builder is not None and prev_builder != full_latest_file:
-            logging.warn('Overwriting old {0} builder: {1}'
-                         .format(project_name, prev_builder))
-        m_project['target']['file'] = full_latest_file
-        _write_version_file(project_name, latest_file)
-    except KeyError as ke:
-        logging.error('FATAL: Formatting issue encountered in manifest. '
-                      'Exiting. \n{0}'.format(ke))
-        sys.exit(1)
+def _publish_latest(builder_dir):
+    for file in glob.glob(os.path.join(builder_dir, '*.json')):
```

Comment:
Since these are already checked in as <runtime>.json, I figured I would just leave them the way they are, unless you strongly prefer that I change them